test/gtest/ucp/test_ucp_peer_failure.cc

```diff
@@ -570,3 +573,167 @@ UCS_TEST_P(test_ucp_peer_failure_keepalive, kill_receiver,
 }
 
 UCP_INSTANTIATE_TEST_CASE(test_ucp_peer_failure_keepalive)
+
+
+struct test_rndv_abort : public test_ucp_peer_failure {
```

Comment:
let's rename to `test_ucp_peer_failure_rndv_abort`, naming is useful for `--gtest_filter` when need to test err flows

Reply:
Done.